apps/emqx_authn/src/emqx_authn_app.erl

```diff
@@ -34,6 +34,7 @@
 start(_StartType, _StartArgs) ->
     ok = mria_rlog:wait_for_shards([?AUTH_SHARD], infinity),
     {ok, Sup} = emqx_authn_sup:start_link(),
+    ok = emqx_authn_utils:cleanup_resources(),
```

Comment:
This cleanup doesn't make any difference now, but I added it so that we preserve it when fix `emqx_authentication` reinitialization.